canu/utils/sls_utils/Networks.py

```diff
@@ -251,6 +251,19 @@ def __init__(self, default_route_network_name="CMN"):
         self.__system_default_route = default_route_network_name
         self.mtu(network_mtu=9000)
 
+    def system_default_route(self, default_route=None):
+        """Retrieve and set the default route for the system.
+
+        Args:
+            default_route (str): CHN, CAN, or CMN
+
+        Returns:
+            name (str): Short name of the network for the getter
+        """
+        if default_route is not None:
+            self.__system_default_route = default_route
+        return self.__system_default_route
```

Comment:
So running this without args retrieves the route, running it with args sets the route.

I'd suggest using accessors (`@property` instead, defining `self.__system_default_route` earlier.

Branching and making a quick example.

Reply:
Example.. https://github.com/Cray-HPE/canu/compare/CASMNET-2319-sls-utils-system-default-route...CASMNET-2319-sls-utils-system-default-route-rusty?expand=1

Come to think of it, it's tempting for me not go nuts and replace most of these class's functions with accessors.